HW_1/task04.py

```diff
@@ -0,0 +1,12 @@
+# Задача 4: Петя, Катя и Сережа делают из бумаги журавликов. Вместе они сделали S журавликов. Сколько журавликов сделал каждый ребенок, если известно, 
+# что Петя и Сережа сделали одинаковое количество журавликов, а Катя сделала в два раза больше журавликов, чем Петя и Сережа вместе?
+# *Пример:*
+# 6 -> 1  4  1
+# 24 -> 4  16  4
+# 60 -> 10  40  10
+
+bird = int(input("Введите кол-во журавликов: "))
+katya = int((bird/3)*2)
+petya = int((katya/2)/2)
+serega = int(petya)
```

Comment:
Место функции `int` можно использовать операторы целочисленного отделения. К тому же было бы неплохо проверять входные данные на кратность 6. Операции деления и  умножения имеют одинаковый приоритет. В Python, да и в других языках программирования не принято использовать в качестве имен переменных nранслитерацию русских слов.
```suggestion
katya = bird // 3 * 2
petya = serega = katya // 2 // 2
```